### PR TITLE
Show of the week

### LINKED ISSUE #6: As a DJ, I want my show to be featured, so it gets more exposure.
FeaturedShow

Responsibilities

- The featured show
- Info about the show

Collaborators

- Exec member
- Show

### PR DESCRIPTION
**Pull request by schester77**
Created "ShowOTW" component and added it to "Home" page in index.js.

### CODE REVIEW

File: src/components/ShowOTW.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { render, screen } from "@testing-library/react";
+import ShowOTW from "./ShowOTW.js"; 
+
+const sampleShow = {
```
Comment (ethansaxenian) [inline src/components/ShowOTW.test.js:4]: At some point we could probably move the sample show into `globals.js`, or even create a new file called `test-utils.js` or something. Definitely not crucial right now though.

**Review by ethansaxenian — APPROVED** [review]
Looks great!

**Review by ethanberner — APPROVED** [review]
Looks great! 2/3

**Review by maddievb — APPROVED** [review]
looks great! 3/3

**Pull request merged**